packages/core/src/browser/style/view-container.css

```diff
@@ -38,14 +38,24 @@
 
 .theia-view-container > .p-SplitPanel > .p-SplitPanel-handle:after {
     background-color: var(--theia-sideBarSectionHeader-border);
-    min-height: 2px;
-    min-width: 2px
+    min-height: var(--theia-sash-width);
+    min-width: 2px;
 }
 
 .theia-view-container > .p-SplitPanel > .p-SplitPanel-handle {
     background-color: var(--theia-sideBarSectionHeader-border);
 }
 
+.p-SplitPanel > .p-SplitPanel-handle:hover:after {
+    background-color: var(--theia-sash-hoverBorder);
+    transition-delay: var(--theia-sash-hover-delay);
+    min-width: var(--theia-sash-width);
```

Comment:
Minor: there's a very short fade out transition when hovering: The style in VSCode looks like this:
`transition: background-color .1s ease-out`